### ISSUE TITLE
RouterOS v7.6 wireless_clients API format

### ISSUE THREAD

**Issue by rovitsky** [akpw/mktxp#84]
I have two MikroTik routers with RouterOS v7.6: hAP ax3 (C53UiG+5HPaxD2HPaxD) and hAP ax2 (C52iG-5HaxD2HaxD). Exporter is hosted in docker on Raspberry Pi 4 (Ubuntu 22.04.2 with all packages upgraded to latest versions), used image is "ghcr.io/akpw/mktxp:gha-4465496420".

If **wireless_clients** flag is set to False, exporter works just fine, all data is in the response. But if I set **wireless_clients** to True (for at least one of the router configurations), only python metrics are returned by exporter and in logs there is this error:
> Exception occurred during processing of request from ('192.168.0.15', 59839)
Traceback (most recent call last):
File "/usr/local/lib/python3.11/socketserver.py", line 317, in _handle_request_noblock
  self.process_request(request, client_address)
File "/usr/local/lib/python3.11/socketserver.py", line 348, in process_request
  self.finish_request(request, client_address)
File "/usr/local/lib/python3.11/socketserver.py", line 361, in finish_request
  self.RequestHandlerClass(request, client_address, self)
File "/usr/local/lib/python3.11/socketserver.py", line 755, in __init__
  self.handle()
File "/usr/local/lib/python3.11/http/server.py", line 432, in handle
  self.handle_one_request()
File "/usr/local/lib/python3.11/http/server.py", line 420, in handle_one_request
  method()
File "/usr/local/lib/python3.11/site-packages/prometheus_client/exposition.py", line 270, in do_GET
  status, headers, output = _bake_output(registry, accept_header, accept_encoding_header, params, False)
File "/usr/local/lib/python3.11/site-packages/prometheus_client/exposition.py", line 105, in _bake_output
  output = encoder(registry)
File "/usr/local/lib/python3.11/site-packages/prometheus_client/exposition.py", line 198, in generate_latest
  for metric in registry.collect():
File "/usr/local/lib/python3.11/site-packages/prometheus_client/registry.py", line 97, in collect
  yield from collector.collect()
File "/usr/local/lib/python3.11/site-packages/mktxp/flow/collector_handler.py", line 125, in collect
  yield from self.collect_sync()
File "/usr/local/lib/python3.11/site-packages/mktxp/flow/collector_handler.py", line 44, in collect_sync
  yield from collect_func(router_entry)
File "/usr/local/lib/python3.11/site-packages/mktxp/collector/wlan_collector.py", line 55, in collect
  BaseOutputProcessor.augment_record(router_entry, registration_record)
File "/usr/local/lib/python3.11/site-packages/mktxp/flow/processor/output.py", line 55, in augment_record
  registration_record['tx_rate'] = BaseOutputProcessor.parse_bitrates(registration_record['tx_rate']) \
File "/usr/local/lib/python3.11/site-packages/mktxp/flow/processor/output.py", line 112, in parse_bitrates
  rate = int(rate)
ValueError: invalid literal for int() with base 10: '433.0Mbps'

I've made some changes to /flow/processor/output.py file to disable check for wifiwave2 and explicitly set ww2_installed to False at line 53
`#ww2_installed = WirelessMetricsDataSource.wifiwave2_installed(router_entry)`
`ww2_installed = False`
and injected the file into container mounting it as a volume. I can confirm that this approach worked for me, but I guess it is impossible to change this in repository, because there might be some older RouterOS versions (or whatever is providing the value) out there that use XXX.XMbps format instead of int for wifiwave2.

I'm not extremely familiar with the MikroTik API, so I wouldn't dare make any changes here right now, but maybe this issue will draw @akpw's attention or at least provide a workaround for someone like me

**Comment by akpw** [comment]
hi @rovitsky, thanks for the report. A quick question:
>for at least one of the router configurations
can you pls provide more info on which of the two routers it was, and does it have the wifiwave2 package installed?

I provided a quick workaround in the meantime, pls feel free to try out in the [latest](https://github.com/akpw/mktxp/pkgs/container/mktxp)

**Comment by rovitsky** [comment]
Yeah, my bad, @akpw, I could've phrased this better. What I meant was that if **either** of configs had wireless_clients flag set the whole response for both routers would not be constructed properly, probably because that exception was not caught. Both routers have wifiwave2 package installed.

Before the fix I've tried different configurations
1) no router conf had wireless_clients set - metrics were there, but obviously without wireless clients data
2) one (tried with both just for good measure) - metrics were missing even for the router, that had wireless_clients False in conf
3) both configs had flag set - no metrics for either router

As for the fix - thank you very much, it works like a charm. I've just deployed it without injecting my "fixed" file and all metrics are there.
I also have a **RB951G-2HnD** which doesn't have wifiwave2 installed, but that one is not covered by monitoring. I'll try to deploy exporter for it to see if it can pull any data and get back to you with the results. I guess it would be nice to know that non-wifiwave2 devices are not broken by the latest changes
